src/App.tsx

```diff
@@ -1,61 +1,103 @@
 import './App.scss';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { Todo } from './react-app-env';
 
 export const App = () => {
+  const [todos, setTodos] = useState(todosFromServer);
+  const [title, setTitle] = useState('');
+  const [selectUser, setSelectUser] = useState(0);
+  const [submit, setSubmit] = useState(false);
+
+  const largestId = Math.max(...todos.map(todo => todo.id));
```

Comment:
Calculate it only when you need it (when submitting the form before create new todo)